homeassistant/components/config/entity_registry.py

```diff
@@ -96,6 +97,33 @@ def websocket_get_entity(
     )
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "config/entity_registry/get_entries",
+        vol.Required("entity_ids"): cv.entity_ids,
+    }
+)
+@callback
+def websocket_get_entities(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Handle get entity registry entries command.
+
+    Async friendly.
+    """
+    registry = er.async_get(hass)
+
+    entity_ids = msg["entity_ids"]
+    entries: dict[str, dict[str, Any] | None] = {}
+    for entity_id in entity_ids:
```

Comment:
It looks like this doesn't use the cache here like the other api?

I recall we are already having some performance issues with nodered + other which is being looked at in:
https://github.com/home-assistant/core/pull/84559

Reply:
We can add a cache if this function will be widely used, the plan now is to use for the cloud config panel which needs to know about aliases.

Apart from caching the JSON data, we should perhaps consider not sending the full dicts, both in the endpoint added by this PR and in the existing endpoint `config/entity_registry/get`.

Reply:
let's get it in for now, and if we end up using it in more pages, we should cache it.